frontend/form-builder/src/components/FormSettings/NumberSettings.tsx

```diff
@@ -0,0 +1,27 @@
+import { useTranslation } from "react-i18next";
+import React, { useContext } from "react";
+import { FormBuilderContext } from "../../context";
+import { Textfield } from "@digdir/design-system-react";
+
+export const NumberSettings = () => {
+  const { t } = useTranslation();
+  const { selectedItem } = useContext(FormBuilderContext);
+  return (
+    <>
+      <Textfield
+        name="greaterThan"
+        label={t("settings_side_bar.minimum.value")}
+        defaultValue={selectedItem.greaterThan || ""}
+        size="small"
+        placeholder="Optional"
+      />
+      <Textfield
+        name="lessThan"
+        label={t("settings_side_bar.maximum.value")}
+        defaultValue={selectedItem.lessThan || ""}
+        size="small"
+        placeholder="Optional"
+      />
```

Comment:
```suggestion
  const { greaterThan = "", lessThan = "" } = useContext(FormBuilderContext).selectedItem ;
  return (
    <>
      <Textfield
        name="greaterThan"
        label={t("settings_side_bar.minimum.value")}
        defaultValue={greaterThan}
        size="small"
        placeholder="Optional"
      />
      <Textfield
        name="lessThan"
        label={t("settings_side_bar.maximum.value")}
        defaultValue={lessThan}
        size="small"
        placeholder="Optional"
      />
```
This is just one optional solution using destructuring. Both solutions work fine! :)

Reply:
Nice, I did not know that you could insert a "fallback value" into the destructuring. I went with this solution 🙂